platform_versioned_docs/version-23.3.0/enterprise/prerequisites/azure.mdx

```diff
@@ -7,182 +7,310 @@ tags: [azure, prerequisites, configuration]
+  ```
 
-1. Enter **Public IP addresses** in the search.
+  The `sku-name`, `tier`, `storage-size`, and `iops` values depend on your performance requirements.
 
-2. Under **Services**, select **Public IP addresses**.
+  1. Run `az mysql flexible-server db create` to create a database on your server:
 
-3. On the **Public IP addresses** page, select the entry containing your VM name. A page opens with that IP's details.
+  ```console 
+  az mysql flexible-server db create --resource-group towerrg
+                                   --server-name towerdbserver
+                                   --database-name towerdb
+  ```                                
 
-4. Select **Configuration** from the left-hand navigation panel.
+</details>
 
-5. Ensure that your IP address assignment is **Static**.
+After your database is created, update your Seqera [configuration](../configuration/overview.mdx#seqera-and-redis-databases) with the database hostname, Admin username, and password.
 
-6. Do not add a custom DNS name label to the VM.
+:::note
+When creating a MySQL user, use the `USER@HOSTNAME` format for the `TOWER_DB_USER` environment variable. For Azure managed MySQL, it's [recommended][azure-db-config] to pass an explicit `serverTimezone` to the `TOWER_DB_URL` environment variable, which (depending on your configuration) may be `UTC`. The DB connection string should be similar to `jdbc:mysql://towerdbserver.mysql.database.azure.com/towerdb?serverTimezone=UTC`.
+:::
 
-To allow ingress on port 8000:
+### Azure Linux VM
 
-1. Enter **Virtual Machines** in the search bar.
+Create a VM instance with these attributes:
+
+- Use **default values** unless otherwise specified.
+- At least **2 CPUS** and **8GB RAM**.
+- **Ubuntu Server 22.04 LTS - Gen2** image.
+- **Accessible by SSH**.
+- Implement DNS or load balancing via [Azure Application Gateway][azure-dns].
+
+Create an Azure Linux VM:
+- [Via the Azure portal][azure-linux-vm-portal]
+- [Via the Azure CLI][azure-linux-vm-cli]
+
+<details>
+  <summary>Create a VM via Azure portal</summary>
+
+  1. Under **Basics**, select your **Subscription** and **Resource group**.
+  1. Under **Instance details**:
+      - Enter a **VM name**
+      - Select the same **Region** as your resource group.
+      - Select the **Ubuntu Server 20.04 LTS - Gen2** image.
+      - Do not set the VM as an **Azure Spot instance**.
+      - Select the **Size** — B2ps v2 or higher is recommended.
+  1. Under **Administrator account**:
+      - Select **SSH public key**
+      - Enter a **username**
+      - Select **Generate new key pair**
+      - Enter a **Key pair name**
+  1. Under **Inbound port rules**:
+      - Select **Allow selected ports**
+      - Select **SSH (22)** and **HTTP (80)** from the dropdown
```

Comment:
Depending on the docker-compose file, the ports 443 and 8000 are needed here too. They need to open port 443 if they want SSL/TLS termination.